go/test/endtoend/vreplication/multi_tenant_test.go

```diff
@@ -98,6 +101,30 @@ const (
     "t1": {}
   }
 }
+`
+	mtShardedVSchema = `
+{
```

Comment:
Doesn't get much nittier than this, but it looks like we have a mix of spaces and tabs in this value (and maybe a mix of 2 spaces and 4 for indentation).